pkg/operation/botanist/secrets.go

```diff
@@ -548,10 +548,10 @@ func (b *Botanist) generateWantedSecrets(basicAuthAPIServer *secrets.BasicAuth,
 		},
 	})
 
-	// Secret definition for kubecfg-internal
+	// Secret definition for kubecfg-internal-probe
 	secretList = append(secretList, &secrets.ControlPlaneSecretConfig{
 		CertificateSecretConfig: &secrets.CertificateSecretConfig{
-			Name:      common.KubecfgInternalSecretName,
+			Name:      common.KubecfgInternalProbeSecretName,
```

Comment:
I think that a clean-up logic is required for the next couple of releases. It is needed to delete the obsolete `kubecfg-internal` secret.

Here is an example for clean-up of deprecated resources https://github.com/gardener/gardener/blob/80371ded41e91be51cd3253342363571a73706be/pkg/controllermanager/controller/project/project_control_reconcile.go#L150-L167

Reply:
The clean-up logic is already present [here](https://github.com/gardener/gardener/pull/1734/files#diff-5ca37df1e48dc263572bf7822db58a8aR763).

But this clean-up logic must be removed in one of the later releases, of course.

Reply:
Nice.
Excuse me for the misunderstanding.

Reply:
Thanks! No worries 🙂